src/main/kotlin/uk/co/craigbass/pratura/usecase/CurrencySetter.kt

```diff
@@ -0,0 +1,7 @@
+package uk.co.craigbass.pratura.usecase
+
+import uk.co.craigbass.pratura.domain.Currency
```

Comment:
I'd be inclined to shove the Retriever and Setter together in one file

I've just realised these are both named after breeds of dog 🐶